package.json

```diff
@@ -14,13 +14,21 @@
     "next": "13.4.19",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "typescript": "5.2.2",
-    "sitefinity-react-framework": "file:./framework/build/sitefinity-react-framework-0.1.0.tgz"
+    "sitefinity-react-framework": "file:framework/build/sitefinity-react-framework-0.1.0.tgz",
```

Comment:
With this here do we need the explicit install in the ci step?

Reply:
no more file:framework/build/sitefinity-react-framework-0.1.0.tgz